instrumentation/httpclient5/src/main/java/brave/httpclient5/HttpClient5Tracing.java

```diff
@@ -43,15 +43,19 @@ public static HttpClient5Tracing newBuilder(HttpTracing httpTracing) {
     return new HttpClient5Tracing(httpTracing);
   }
 
+  public HttpClientBuilder addTracingToBuilder(HttpClientBuilder builder) {
```

Comment:
@gmellemstrand I believe, adding additional method like this would introduce conflicting  (`addTracingToBuilder` vs `build`) instrumentation and confusion (when `addTracingToBuilder` should be used and why). I would recommend to have a factory method instead, fe:

```suggestion
    public static HttpClient5CustomTracing newBuilder(HttpTracing httpTracing, HttpClientBuilder builder) {
      return new HttpClient5CustomTracing(httpTracing, builder);
    }
```

In this case the `HttpClient5CustomTracing` type would take care of adding tracing to the provided `HttpClientBuilder` as well as having a `build()` method with no args. 

@codefromthecrypt does it make sense to you?